src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -62,6 +62,44 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         } private set;
     }
 
+    /*********************************************************************************************************
+    * @description Override saveSettings from STG_Panel to update Custom Setting Error Setting
+    * @return PageReference 
+    */
+    public override PageReference saveSettings() {
+        STG_SettingsManager_CTRL.idPanelCurrent = idPanel();
+        Savepoint sp = Database.setSavepoint();
+        try {
+            List<SObject> toUpdate = new List<SObject>();
+
+            if (STG_Panel.stgService.stgErr != null && !Test.isRunningTest()) {
+                toUpdate.add(STG_Panel.stgService.stgErr);
+            }
+
+            Boolean hasUpdatePermission = UTIL_Describe.getObjectDescribe(
+                String.valueOf(Error_Settings__c.SObjectType)
+            ).IsUpdateable();
+
+            if (!hasUpdatePermission) {
+                throw new DmlException(
+                    String.format(
+                        System.Label.flsError,
+                        new List<String>{System.Label.stgLabelErrorSettings})
+                );
+            }
+
+            update toUpdate;
+            listDR = new List<DetectResult>();
+
+        } catch (Exception e) {
+            Database.rollback(sp);
+            ERR_Handler.processError(e, ERR_Handler_API.Context.STTG);
```

Comment:
Will `ERR_Handler.processError` add error message to the VF page? Check that only one error message is added to the VF page.

Reply:
`ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.ERROR, e.getMessage()));`(next line) adds the exception message to the error page. ERR_Handler.processError creates Error records in the error log.

Reply:
Confirmed that only 1 message will be displayed onto the page and 1 error record will be created in error log.